schema/factory_builder.go

```diff
@@ -96,10 +97,14 @@ func (fb *FactoryBuilder) deserializeProperties(properties jsoniter.RawMessage,
 		dec := jsoniter.NewDecoder(bytes.NewReader(v))
 		dec.UseNumber()
 		if err = dec.Decode(&builder); err != nil {
+			if strings.Contains(err.Error(), ErrInvalidType.Error()) ||
```

Comment:
do we need contains as you're returning an exact error?

Reply:
I tried errors.Is and it doesn't work by some reason.
The exact error looks like this:
```
schema.FieldBuilder.TypeRaw: unmarshalerDecoder: only one non-null type allowed for the field, error found in #10 byte of ...|,\"object\"]}|..., bigger context ...|{\"type\":[\"null\",\"string\",\"object\"]}|...
```

Reply:
Actually yes, the unmarshaler will change the error.